### DISCUSSION TITLE
PostgreSQL or MariaDB

### DISCUSSION THREAD

**Discussion by TommyTran732** [dani-garcia/vaultwarden#2931] [General]
Hi,

I wonder if there is any difference in terms of support for PostgreSQL and MariaDB? I am trying to decide which database to use between the 2.

**Comment by BlackDex** [comment]
Support in which sense?
Vaultwarden isn't going to support database setup or issues it self, unless there is a query issue or something of course.
But if you mean features in the sense of Vaultwarden/Bitwarden features, there is no difference between any of the 3 database types which Vaultwarden is able to talk to.

**Reply by TommyTran732** [reply]
Support as in having fewer issues with these databases as time go on / getting more attention of the devs

**Reply by BlackDex** [reply]
That is still very broad. And for the future that is hard to tell for any of us.
I would just say to pick a DB which you are most comfortable with.

We try to keep supporting all the databases we currently support.

**Reply by TommyTran732** [reply]
Yeah, I just don't wanna run into problems with future database versions and what not. But thanks :p
